src/utils/locale.ts

```diff
@@ -245,105 +245,114 @@ function getLocale(component: LocalizedComponent): string {
   );
 }
 
-interface NumberStringFormatOptions {
+interface NumberStringFormatOptions extends Intl.NumberFormatOptions {
   numberingSystem: NumberingSystem;
   locale: string;
-  useGrouping: boolean;
 }
 
 /**
  * This util formats and parses numbers for localization
  */
 class NumberStringFormat {
-  numberingSystem: string;
-
-  locale: string;
-
-  useGrouping: boolean;
-
   /**
-   * The actual group separator for the specified locale
-   * some white space group separators don't render correctly in the browser
-   * so we replace them with a normal <SPACE>
+   * The actual group separator for the specified locale.
+   * Some white space group separators don't render correctly in the browser,
+   * so we replace them with a normal <SPACE>.
    */
-  actualGroup: string;
+  private _actualGroup: string;
 
   /** the corrected group separator */
-  group: string;
+  private _group: string;
+
+  public get group(): string {
+    return this._group;
+  }
+
+  private _decimal: string;
+
+  public get decimal(): string {
+    return this._decimal;
+  }
+
+  private _minusSign: string;
 
-  decimal: string;
+  public get minusSign(): string {
+    return this._minusSign;
+  }
+
+  private _digits: Array<string>;
+
+  public get digits(): Array<string> {
```

Comment:
Nitpick: `public` is the default access modifier, so not necessary.

Reply:
Tsk Tsk, VSCode's auto completion should know better